tests/test_math.hh

```diff
@@ -48,6 +50,7 @@ namespace rascal {
       ref_file >> ref_data;
       unit_vectors = ref_data.at("unit_vectors").get<StdVector2Dim_t>();
       harmonics = ref_data.at("harmonics").get<StdVector3Dim_t>();
+      // a vector which contains Switzerland's mountain landscape
```

Comment:
Ok, I was already wondering what `alps` was referring to from the new added reference data....
I thought it was some weird index combination like in other places `a_n_l_m` thing, just differently, but they are the actual alps...nice.

Reply:
Always nice to look out my window and see the snow-capped Associated Legendre PolynomialS.